OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs

```diff
@@ -222,7 +294,7 @@ public ServerListLogic(Widget widget, ModData modData, Action<GameServer> onJoin
 			var playersLabel = widget.GetOrNull<LabelWidget>("PLAYER_COUNT");
 			if (playersLabel != null)
 			{
-				var playersText = new CachedTransform<int, string>(c => c == 1 ? "1 Player Online" : c.ToString() + " Players Online");
+				var playersText = new CachedTransform<int, string>(p => modData.Translation.GetString(PlayersOnline, Translation.Arguments("players", p)));
```

Comment:
This is called every frame

Reply:
No, this is created once in the constructor.